README.md

```diff
@@ -142,7 +141,7 @@ def _adjust_s3_bucket(
     s3_bucket: str,
 ) -> None:
     """Set bucket to a test one."""
-    settings.AWS_STORAGE_BUCKET_NAME = s3_bucket
+    default_storage.bucket_name = s3_bucket
```

Comment:
Is it change really needed? Usually django settings changed in `conftest.py` files (but using `pytest_configure` hook)

Reply:
Since we grabbing bucket name from fixture we can't do it in pytest configure. Also at this point settings for storage are already set, and that means changing AWS_STORAGE_BUCKET_NAME won't do anything.